src/components/typography/text-sample.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Typography } from '@mui/material';
+import * as React from 'react';
+
+const LETTERS = 'AaBbCcDDEEFFGgHHIiJjKKLLMmNNOoPPQqRRSsTtUuVvWwXxYyZz';
+const NUMBERS = '0123456789';
+
+function capitablize(s: string) {

```

Comment:
Nit: spelling.  Also - why is this necessary?  Do different places define/require different variations (capitalized/toUpperCase/neither)?

Reply:
- This function has been removed
- Add the src/constants.ts file